hr_cae/views/hr_employee.xml

```diff
@@ -124,19 +132,16 @@
                     <group name="contribution" string="Flat Contribution">
                         <field name="contribution_date_start"/>
 <!-- todo                            s_fixed_contribution_advance = modalité d'apport-->
-                        <field name="contribution_exemption_ids"/>
-                    </group>
-                </page>
-                <page name="documents" string="Documents">
-                    <group name="passport" string="Passport">
-                        <field name="passport_id"/>
-                    </group>
-                    <group string="Work Permit" name="cae_work_permit">
-                        <field name="visa_no"/>
-                        <field name="permit_no"/>
-                        <field name="visa_expire"/>
+                        <field name="contribution_exemption_ids">
+                            <tree editable="bottom">
```

Comment:
Why did you specify the tree here (its the same as the normal tree for this model)? To make them editable? Can't this be done in another way?

Reply:
To make it editable indeed. Not sure there is another way.